README

```diff
@@ -731,7 +729,7 @@ Disable with 0
 Default value is 10
 
 XLIO_SELECT_SKIP_OS
-Similar to XLIO_RX_SKIP_OS, but in select() or poll() this will force the XLIO
+In select(),poll() or epoll() this will force the XLIO
 to check the non offloaded fd even though an offloaded socket has ready
 packets found while polling.
```

Comment:
make a pretty layout with up to 80 symbols in a row

Reply:
Correct layout ported from vNext